src/mixxx.cpp

```diff
@@ -1148,6 +1153,21 @@ void MixxxMainWindow::connectMenuBar() {
         m_pMenuBar->onNumberOfDecksChanged(m_pPlayerManager->numberOfDecks());
     }
 
+    if (m_pTrackCollectionManager) {
+        connect(m_pMenuBar,
```

Comment:
There is no need to take care of this here or add any code comments, but I think we should get this out of the menu bar and into the library GUI at some point. Then we can work towards finally getting rid of the menu bar.